CONTRIBUTING.md

```diff
@@ -64,10 +64,10 @@ yarn format:scss
 
 ## Playground
 
-We provide a playground in this repository that hot-reloads on change. The playground points to a remote MeiliSearch server that contains the scraped [documentation of MeiliSearch](https://docs.meilisearch.com/).
+We provide a playground in this repository that hot-reloads on change. The playground points to a remote Meilisearch server that contains the scraped [documentation of Meilisearch](https://docs.meilisearch.com/).
 You can play arround with the options and or just try out the search.
 
-For example, type `create an indxe` to experience MeiliSearch [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
+For example, type `create an indxe` to experience Meilisearch [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
```

Comment:
```suggestion
For example, type `create an index` to experience Meilisearch [typo tolerance](https://docs.meilisearch.com/reference/under_the_hood/typotolerance.html).
```

Reply:
It is on purpose to showcase the typo tolerancy 😊